.gitignore

```diff
@@ -98,4 +98,4 @@ integration_tests/sdk/test-credentials.yml
 integration_tests/sdk/test-config.yml
 integration_tests/sdk/test-config-run.yml
 
-
+**/.devcontainer/
```

Comment:
Typo?

Reply:
I added to exclude .devcontainer in .gitignore, as I normally develop in containers.  I'm not sure what team members use here, if they plan to use dev containers, we can standardize on that, it makes it easy. In getting my test environment to work natively on Mac, I had to manually install many python packages.  Please let me know if there is a concern in the approach.

Reply:
Ah I see so it's not a unintentional commit. I'm fine with this then!